### PR TITLE
RPC validators calls IncrementAccum if necessary

### PR DESCRIPTION
**Pull request by jaekwon**
This isn't a big deal, but good to fix nevertheless.

* [n/a] Updated all relevant documentation in docs
* [x] Updated all code comments where relevant
* [x] Wrote tests
* [x] Updated CHANGELOG_PENDING.md

### CODE REVIEW

**Comment by melekes** [comment]
This needs to be rebased against develop.

**Comment by melekes** [comment]
Rebased.

File: state/store.go [hunk 1]
```diff
@@ -191,12 +193,14 @@ func LoadValidators(db dbm.DB, height int64) (*types.ValidatorSet, error) {
 				),
 			)
 		}
+		valInfo2.ValidatorSet.IncrementAccum(int(height - valInfo.LastHeightChanged)) // mutate
```
Comment (melekes) [inline state/store.go]: Should we write a test for this?

**Comment by ebuchman** [comment]
Can we please get a quick explanation of how this was found / what it fixes and a corresponding test? Thanks!

**Comment by melekes** [comment]
I think this was discovered by one of the validators. When we were loading vals, we didn't incrementAccum, so the accum values were incorrect.

```
I found out that in 0.26, the accum values in validators endpoint from 26657 are not updated by each block. Therefore, the value is not useful to inform me the exact accum situation of the whole validators. Although in dump_consensus_state endpoint, it provides current accum status, the historical data is not available. 
```

**Comment by melekes** [comment]
wrote a test (TestStoreLoadValidatorsIncrementsAccum)

before
```
--- FAIL: TestStoreLoadValidatorsIncrementsAccum (0.00s)
        Error Trace:    state_test.go:282
        Error:          Should not be: -10
        Test:           TestStoreLoadValidatorsIncrementsAccum
        Messages:       expected Accum value to change between heights
FAIL
exit status 1
FAIL    github.com/tendermint/tendermint/state  0.055s
```

after:
```
PASS
ok      github.com/tendermint/tendermint/state  0.050s
```

File: state/store.go (hunk 1 above)
Comment (ebuchman) [inline state/store.go]: What happens if LoadValidators is called multiple times before the validator set changes? Won't we keep incorrectly changing the accum?

Comment (ebuchman) [reply state/store.go]: You're right, the result of LoadValidators doesn't seem to be saved anywhere. Excellent! This looks fine to merge then.

**Pull request merged**